src/com/madalinb/coffeeshop/tests/CoffeeShopTest.kt

```diff
@@ -0,0 +1,34 @@
+package com.madalinb.coffeeshop.tests
+
+import com.madalinb.coffeeshop.data.CoffeeShop
+import org.junit.Before
+import org.junit.Test
+
+class CoffeeShopTest {
+    var coffeeShop: CoffeeShop? = null
+
+    @Before
+    fun setUp() {
+        coffeeShop = CoffeeShop("Starbucks Seattle", 47.5809, -122.3160)
+    }
+
+    @Test
+    fun distanceTo() {
+        assert(coffeeShop?.distanceTo(47.6, -122.4) == 0.0861441234211632)
+    }
+
+    @Test
+    fun getName() {
```

Comment:
By looking at the name of the test, i don't have much context of what it actually verifies. 

https://stackoverflow.com/questions/155436/unit-test-naming-best-practices

Reply:
Changed the name to a more appropriate one.